### ISSUE TITLE
💡 Pin the first message in thread

### ISSUE THREAD

**Issue by ririko5834** [MarcusOtter/discord-needle#56]
Automatically pin the first message sent by bot in a thread so you can go to jump to it easily using pinned messages when you want to archive thread.

**Comment by MarcusOtter** [comment]
Yes! I would very much like to do this again. The bot actually had that in the past (before v1.0.0) but I removed it since I thought it was a bit too spammy with this message: 
![image](https://user-images.githubusercontent.com/35617441/152385410-b6162e63-344e-4a89-aafa-cd6515b0f97a.png)

However, I realized yesterday that this is one of the few system messages that you are allowed to delete. Definitely going to add this back, thank you for bringing it up!
